### ISSUE TITLE
standby/energy saving

### ISSUE THREAD

**Issue by GralfR** [rescuezilla/rescuezilla#173]
I've the ISO 2.1.2 and finally got my image backup running. The status says it will take about 90min.
Suddenly the display went black and I was afraid the system crashed. But it was just some kind of energy saving/display blanking and I got my status back by moving the mouse.

I wonder if this is neccesary. For me, I'd like to have a look at the status whenever I like. I'm doing other tasks (like writing issues/requests on github) during long processes. If the display changes I think a new GUI opened or there is something wrong.

In short: is it possible to disable the energy savings, display blanking and screensaver for the live ISO of rescuezilla by default? Best way to save energy is to switch off the display or close the lit of a laptop. In that case I do not want the system to shutdown while doing a backup.

Maybe there could be an option inside the boot-menu: "rescuezilla - no energy saving" or something like that.

What do others think about that?

**Comment by shasheene** [comment]
Adding the ability to disable standby / power management is definitely reasonable. I think majority of users benefit from the default behavior with the screen turning itself off automatically, given all it takes to turn the screen back on is simply moving the mouse. Some desktop environments implement the screen _fading_ to black over a few seconds. This is a small usability improvement but it gives users the opportunity to move the mouse, and quickly teaches them that the screen is going into power saving mode.

There are still definitely reasons to disable standby. Ideally this could be done from the Rescuezilla desktop. Such a feature would greatly benefit from a [persistent partition (task #8)](https://github.com/rescuezilla/rescuezilla/issues/8) so the user would only need to configure this setting once rather than every time the system is rebooted.

It sounds like the [kernel boot parameters of `acpi=off` and `apm=off` might be required](https://askubuntu.com/a/1184895) to achieve this goal. But adding a separate boot menu item will not scale. What is really needed is improved support for combinations of boot options, so I have just created [task #177 to support combinations of boot options (eg, graphical fallback mode + load into RAM + no energy saving](https://github.com/rescuezilla/rescuezilla/issues/173).

**The fade to black and ability to disable power saving from _within_ Rescuezilla will provide a better user-experience for newbies. But it's a relatively low priority: the existing default behavior is widely used.**

Until then, you can manually disable power management by starting Rescuezilla, highlighting "English" and pressing Enter then highlighting "Start Rescuezilla" and pressing the 'e' key to open up the editor. Then use the arrow keys to navigate to "quiet splash" and modify it to be "quiet acpi=off apm=off splash". Then pres CTRL-X to boot with those settings. Warning: this likely disables the ability for Rescuezilla your shutdown your computer completely (Remember that old Windows message "It is now safe to shutdown") and other useful things related to power management. I don't recommend doing this, but the option is there if you want.

**Comment by shasheene** [comment]
Oh, by the way, this page may be relevant for you to disable the systemd services associated with auto suspend: https://wiki.debian.org/Suspend#Disable_suspend_and_hibernation

I haven't tested those instructions, but I believe they should work for Rescuezilla to achieve your goals in the short term.

**Comment by GralfR** [comment]
Wow, thanks for the excellant support, shasheene!
I'll try this next time. Maybe that's a solution without losing the shutdown when ACPI/APM is disabled? And maybe just an icon on the desktop or taskbar, which runs this command, is the first workaround for anybody who wants to disable energy saving in rescuezilla live without getting deaper into advanced options on any upcoming bootmenus...?

**Comment by shasheene** [comment]
Everything should be configurable from within Rescuezilla (without booting with special kernel options) with the correct Terminal commands (or after installing GUI program). ACPI (Advanced Configuration and Power Interface) and APM (Advanced Power Management) are just interfaces that the operating system (whether Linux or Windows) can ask to eg, suspend the system.  Sorry for not identifying that was actually what you were after earlier.

As far as I know the only time you'll need to use a kernel option is when there's a hardware or software bug that causes lockups when eg, the power management states are queried etc. This is a very rare situation that few users will ever encounter.

Yeah, adding a desktop icon for power management might be the ideal solution

**Comment by candrapersada** [comment]
Will the screen wake up when the backup and restore is finished or fail automatically?

**Comment by shasheene** [comment]
Rescuezilla v2.1.3 should have the same behavior as a standard Ubuntu system: if the screen goes to sleep it will stay there even after the Rescuezilla operation has completed. If the screen is off simply move the mouse (and maybe, click) then the screen will turn back on again.

**Comment by GralfR** [comment]
Yeah, that's the point. If the screen would turn back on when backup finishes or any error message pops up, that would be okay. If I sit next to the backup waiting for it to complete, I'd like to see its status without any input to the system. For example when using a dual-screen-setup with a KVM-switch working on one system while the other is doing some backup on a live-CD on the second monitor it would be great to get notified if there is a problem or it's finished without always switching the KVM and moving the mouse.
I wonder, where this energysaving-hype comes from. The times, when a screensaver really saved the life of my CRT are over. When I'm working on a system I'd like to have it working. It's so stupid reading a long text or waiting for a tasked to finish and switching the monitor on after the timeout switched it off. That's reducing the lifespan of my monitor more than saving it.

**Comment by shasheene** [comment]
I'll try and add a notification bell sound when an operation completes. I'll see if there's a way to wake up the monitor too

**Comment by GralfR** [comment]
Great! The notification-sound should also use the PC speaker, because most desktops do not have a speaker attached to the soundcard when doing troubleshooting with an open case on top of the desk.
Regarding wake up the monitor by software: This is possible, because there are tools out there which exactly promise to do that: [no sleep](https://www.cnet.com/how-to/how-to-keep-your-computer-awake-without-touching-the-mouse/)

**Comment by candrapersada** [comment]
> Regarding wake up the monitor by software: This is possible, because there are tools out there which exactly promise to do that: [no sleep](https://www.cnet.com/how-to/how-to-keep-your-computer-awake-without-touching-the-mouse/)

the information you give for windows is not for linux.

**Comment by shasheene** [comment]
Relevant discussion for GParted Live: https://gitlab.gnome.org/GNOME/gparted/-/issues/149

**Comment by PeterGyger** [comment]
Hello 
2024 and the Problem not solved. I can't use this tool, when the Montior switch off.
Solution? Thanks

**Comment by GralfR** [comment]
> I might've found a solution.
> 
> 4. `apt install xfce4-screensaver xfce4-settings`

Not a solution in any case. There are areas around with no internet connection. How to install anything without internet?
And in addition: why do I have to install it each time? There is a second problem: no persistance with any settings using live-ISO.
So, if the solution is that easy, please just include that update into the ISO.

**Comment by candrapersada** [comment]
and add touchpad setting

**Comment by PeterGyger** [comment]
Hello
The installation of the screensaver menu has worked.
I have switched it off there.
However, the monitor is still deactivated.
This means that the monitor shows that there is no signal on HDMI1. No matter whether I click with the mouse or make an entry on the keyboard. The monitor remains switched off.

I can reproduce this effect on 3 computers and two monitors. This means that I have to restart the computer using the on/off switch. This is not OK for me. I can't use the tool like this

Greetings

**Comment by JavaJava2Java** [comment]
You guys gave me a standard idea I didn't think thru, I will try the following:

From:
https://askubuntu.com/questions/1217597/mousemover-to-prevent-idle

There's a program that moves the mouse when it detects you are away, or idle.

https://github.com/carrot69/keep-presence/

You can install it from snap:

sudo snap install keep-presence

Then run it:

keep-presence --seconds 30

It will move the mouse after 30 seconds if it detects that you are idle.
-------------------------------
Let you know how that works for me. Thanks again to all of you!

**Comment by MaLDox77** [comment]
I can't use rescuezilla because this issue. My PC is connected to a TV that switch off after a few minutes and is not possible to restore image after idle status triggers.

**Comment by kaa1el** [comment]
I downloaded RescueZilla today trying to restore my TV room PC OS SSD drive after it got RMAed from Seagate, after a few minutes the screen went black and I can never wake it up, no matter how hard I try (reconnect HDMI, hitting keyboard and mouse, none worked). Please FIX it ASAP.

**Comment by Vincenz099** [comment]
This is infuriating.

hardware:
Nvidia 4070, AMD 5950x

I start a clone, the stupid resquedisk goes to sleep after x minutes, moving mouse, doing keyboard inputs for a moment maybe woke up the system but no, the screen reports no signal and thats it, no ammount of replugging monitors, inputs or anything will get it out of its dumb sleeping mode, most likely caused by nvida drivers. that are baked into this iso.

How can we use it? how can we disable it? I tried everything in this github history. this tool is 100% unusable like this in the wild, poweroptions should be available, and better yet default disabled.

help please how do i turn off this thing from turning itself off.
